sunpy/image/transform.py

```diff
@@ -325,6 +325,58 @@ def _rotation_skimage(image, matrix, shift, order, missing, clip):
     return rotated_image
 
 
+@add_rotation_function("cv2",
+                       handles_clipping=False, handles_image_nans=False, handles_nan_missing=False)
+def _rotation_skimage(image, matrix, shift, order, missing, clip):
```

Comment:
Should this be named `_rotation_cv2` instead?

Reply:
Ha, whoops, yes.  It technically doesn't matter because the function is never called by name, even internally, so the function names could all be the same, generic name.  But certainly a mismatched name is confusing!